distribution/docs/src/main/resources/content/_managing/_running/ddf-service.adoc

```diff
@@ -41,6 +41,14 @@ ${branding-lowercase}${at-symbol}local> wrapper:install -i setenv-wrapper.conf -
 ----
 ${branding-lowercase}${at-symbol}local> wrapper:install -i setenv-windows-wrapper.conf -n${branding-lowercase} -d ${branding-lowercase} -D "${branding} Service"
 ----
+. Edit `${home_directory}/etc/${branding-lowercase}-wrapper.conf` and uncomment both of these lines
+(remove the `#` at the beginning of the lines):
++
+.${home_directory}/etc/${branding-lowercase}-wrapper.conf
+----
+#wrapper.java.additional.9=-Djava.endorsed.dirs=%JAVA_HOME%/jre/lib/endorsed:%JAVA_HOME%/lib/endorsed:%KARAF_HOME%/lib/endorsed
```

Comment:
✏️ the index value (i.e. 9 and 10) typically needs to be re-numbered appropriately so they are all consecutive